be/src/exec/vectorized/aggregator.cpp

```diff
@@ -252,6 +265,12 @@ Status Aggregator::close(RuntimeState* state) {
         _mem_pool->free_all();
     }
 
+    for (auto ctx : _agg_fn_ctxs) {
```

Comment:
add some comment to explain why ctx->impl->close() is moved here

Reply:
fixed